docs/benchmark.ipynb

```diff
@@ -27,10 +38,30 @@
     "- **Mordred** [5]: Continuous descriptors with more than 1800 2D and 3D descriptors.\n",
     "- **ChemBERTa** [6]: Learned representations from a pre-trained SMILES transformer model.\n",
     "\n",
+    "#### Tl;dr - Use cases\n",
+    "\n",
+    "No featurizer consistently stood out for either task or even within a task category:\n",
+    "\n",
+    "- **Modeling:**  The Mordred featurizer outperforms the next best featurizer by ~about 20% on the _Lipophilicity_ prediction task. On _ClinTox_, however, things are reversed and ChemBERTa outperforms the other featurizers by ~about 18%. This shows the context-dependent nature and thus the importance of experimenting with trying different featurizers.\n",
+    "\n",
+    "- **Search:** ECFP outperforms ChemBERTa and Mordred and is the best option across the board, although it's target dependent.\n",
+    "  \n",
+    "These quick examples show the context-dependent nature and thus the importance of experimenting with trying different featurizers. **In short, the perfect molecular featurizer doesn’t exist (yet!)**. All have their pros and cons depending on the data and the downstream task.\n",
```

Comment:
I would move this up further (above "Use Cases") and / or make it visually distinct. Now it doesn't stand out to me. Maybe add a header? Horizontal line? Colored box?

<details><summary>Screenshot</summary>
<p>

![image](https://user-images.githubusercontent.com/28000055/229373967-1218d98f-be22-4aed-9c3d-e05805e9604b.png)

</p>
</details>